scripts/benchmark.js

```diff
@@ -116,23 +119,54 @@ function test(name, callback) {
   const parseEnd = Date.now();
   const parseNow = parseEnd - parseStart;
 
+  const runTimes = [run];
+
+  for (var i = 1; i < NUM_TEST_RUNS; i++) {
+    const start = Date.now();
+    const result = callback(code);
+    runTimes.push(Date.now() - start);
+  }
+
+  const totalTime = runTimes.reduce((a, b) => a + b, 0);
+  const average = parseInt(totalTime / runTimes.length, 10);
+
   results.push({
     name: name,
     raw: result.length,
     gzip: gzipped.length,
     parse: parseNow,
-    run: run,
+    run: average,
   });
 }
 
 function testFile() {
   code = fs.readFileSync(filename, "utf8");
   gzippedCode = zlib.gzipSync(code);
 
-  test("babili", function (code) {
+  test("babili (best speed)", function (code) {
     return babel.transform(code, {
       sourceType: "script",
-      presets: [require("../packages/babel-preset-babili")],
+      presets: [
+        [require("../packages/babel-preset-babili"), {
+          simplify: {
+            multiPass: false
+          }
+        }]
+      ],
+      comments: false,
+    }).code;
+  });
+
+  test("babili (best size)", function (code) {
+    return babel.transform(code, {
+      sourceType: "script",
+      presets: [
+        [require("../packages/babel-preset-babili"), {
+          simplify: {
+            multiPass: true
```

Comment:
@boopathi this doesn't seem to show any difference in any of the files I'm testing...